packages/forma-36-react-components/tools/.storybook/webpack.config.js

```diff
@@ -16,8 +16,31 @@ module.exports = {
           require.resolve('react-docgen-typescript-loader'),
         ],
       },
+      {
```

Comment:
those fcss classes will be available globally right?

Reply:
Yup! Also good catch - I needed to make this available to production builds. I've updated the PR